recipes/pyxem/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pyXem" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/pyxem/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: f95abf7a1f1353eeedb1d616c1083c05cbca3178d13441e1890d41d26864cd3f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py2k]
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+    - hyperspy
```

Comment:
@ericpre Can we leave it at {{hyperspy}} or does still need to be changed to {{hyperspy-base}}?

Reply:
`hyperspy` will install more packages than `hyperspy-base`, in particular the GUIs packages. Only `hyperspy-base` is a requirement but I guess for the moment, this is more simple for the user to have all these packages installed, so I would leave as it is.
In any case this can be changed very easily latter.